_posts/en/newsletters/2023-10-11-newsletter.md

```diff
@@ -0,0 +1,204 @@
+      programmed, and, more importantly, can only be done at run-time,
+      not compile-time. It satisfies the frequent use case of wanting
+      to take input that can be either kind of identifier. For this reason,
+      this PR doesn't remove `GenTxid`. A better alternative for the
+      future might be `std::variant<Txid, Wtxid>`."
+  a5link="https://bitcoincore.reviews/28107#l-161"
+
+  q6="How is `transaction_identifier` able to subclass `uint256`, given
+      that, in C++, integers are types and not classes?"
+  a6="Because `uint256` is itself a class, rather than a built-in type.
+      (The largest built-in C++ integer type is 64 bits.)"
+  a6link="https://bitcoincore.reviews/28107#l-194"
+
+  q7="Does a `uint256` behave otherwise the same as, for example, a
+      `uint64_t?`"
+  a7="No, arithmetic operations aren't permitted on `uint256` because
+      they don't make sense for hashes (which is the main use of `uint256`).
+      The name is misleading; it's really just a blob of 256 bits.
+      A separate `arith_uint256 ` allows arithmetic (used, for example,
+      in PoW calculations)."
+  a7link="https://bitcoincore.reviews/28107#l-203"
+
+  q8="Why does `transaction_identifier` subclass `uint256` instead of
+      being a completely new type?"
+  a8="It allows us to use explicit and implicit conversions to leave
+      code that is expecting a transaction ID in the form of a `uint256`
+      unchanged until it's an appropriate time to refactor to use the new,
+      stricter `Txid` or `Wtxid` types."
+  a8link="https://bitcoincore.reviews/28107#l-219"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- FIXME:harding to update Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27596][bitcoin core #27596] finishes the first phase
+  of the [assumeutxo][topic assumeutxo] project containing all the
+  remaining changes necessary to both use an assumedvalid snapshot
+  chainstate and do a full validation sync in the background. It
+  makes UTXO snapshots loadable via RPC (`loadtxoutset`) and adds
+  `assumeutxo` parameters to chainparams.
+
+    While the feature set will not be available on mainnet until
+    [activated][bitcoin core #28553], this merge marks the culmination of
+    a multi-year effort. [Proposed in 2018][assumeutxo core dev] and
```

Comment:
Going to change "Proposed in 2018" to "The project was proposed in 2018".  The fundamental idea behind assumeutxo---download only the current chainstate---dates back to early Bitcoin and (it seems to me) may be the idea Nakamoto was trying to work towards in section 7 of his original paper.  After sipa implemented ultraprune (creating the modern chainstate) something like 10 years ago, people have been copying around their chainstates to skip IBD---BTCPay even offers this as a [feature](https://docs.btcpayserver.org/FAQ/Synchronization/#can-i-skip-the-synchronization).

AssumeUTXO as a project took what had been previously proposed and haphazardly implemented and did it in a way that addressed most of the concerns about the idea, making it much more usable and secure.  I think we want to make clear that it's that part which was proposed in 2018, not the much older idea of just copying around the current system state.